zio/vngio/writer.go

```diff
@@ -3,28 +3,10 @@ package vngio
 import (
 	"io"
 
-	"github.com/brimdata/zed/pkg/units"
 	"github.com/brimdata/zed/vng"
 )
 
-const (
-	DefaultColumnThresh = 5 * 1024 * 1024
-	DefaultSkewThresh   = 25 * 1024 * 1024
-)
-
-type WriterOpts struct {
-	ColumnThresh units.Bytes
-	SkewThresh   units.Bytes
-}
-
 // NewWriter returns a writer to w with reasonable default options.
```

Comment:
Since the options have gone away.
```suggestion
// NewWriter returns a writer to w.
```